### DISCUSSION TITLE
Model Checkpoint File

### DISCUSSION THREAD

**Discussion by malsaidi93** [zcemycl/PyTorch-DeepFloorplan#3] [Q&A]
Hi, 
Can you provide checkpoint.pt file for your model? It will be really helpful if you can provide a link to download it or add it to the repository. Thank you!

**Comment by malsaidi93** [comment]
Hi,

I checked readme.md and didn't find the link to download data. If you may
please repost it or send it to me by email and I can check it out.
It is a Very interesting project, I am trying to get more familiar with
this application, if you also have recommendations of work that I can check
out, please let me know.

Thank you for your response, Repo definitely deserves to be starred :)

Best regards,
Mustafa

**Comment by malsaidi93** [comment]
Hi again,

If you can please provide links to both dataset and pretrained model :)

Best regards,
Mustafa

**Comment by zcemycl** [comment]
@malsaidi93 This is a legacy code written 2 years ago, and I actually mistakenly thought this is my tf2 implementation, so may provide you the wrong instruction. 

1. Download tfrecord for training, the link is [here](https://mycuhk-my.sharepoint.com/personal/1155052510_link_cuhk_edu_hk/_layouts/15/onedrive.aspx?id=%2Fpersonal%2F1155052510%5Flink%5Fcuhk%5Fedu%5Fhk%2FDocuments%2Ffloorplan%5Fmodel&ga=1). 
2. Convert tfrecord to csv, then train the model.

I am sorry, I could not provide you the checkpoint/pretrained model since it is lost already. I only have checkpoints/pb/tflite downloadable from TF2DeepFloorplan.

**Comment by malsaidi93** [comment]
Okay thank you :)

Best regards,
Mustafa

**Reply by zcemycl** [reply]
@malsaidi93 Just convert this to discussion, hope it helps others to reuse the code.
